core/src/test/java/org/apache/calcite/test/RelOptRulesTest.java

```diff
@@ -142,6 +142,21 @@ protected DiffRepository getDiffRepos() {
     return DiffRepository.lookup(RelOptRulesTest.class);
   }
 
+  @Test public void testWindowInParenthesis() {
+    HepProgram preProgram =  new HepProgramBuilder()
+        .build();
+
+    HepProgramBuilder builder = new HepProgramBuilder();
+    builder.addRuleClass(ProjectToWindowRule.class);
+    HepPlanner hepPlanner = new HepPlanner(builder.build());
+    hepPlanner.addRule(ProjectToWindowRule.PROJECT);
+
+    final String sql = "select count(*) over (w)\n"
+        + "from emp \n"
+        + "window w as (partition by empno order by empno)";
```

Comment:
please remember not to put a space before the '\n' in queries. Otherwise we have trailing spaces in RelOptRulesTest.xml.

I've fixed it, and also the fact that the sql query in RelOptRulesTest.xml is different to the one in RelOptRulesTest.java.